File: src/main/java/no/nav/syfo/domain/Person.java
package no.nav.syfo.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class Person {
    public String navn;
    public String aktoerId;
    public String fnr;
    public String epost;
    public String tlf;
    public LocalDateTime sistInnlogget;
    public LocalDateTime sisteEndring;
    public LocalDateTime sistAksessert;
    public Boolean samtykke;
}

File: src/main/kotlin/no/nav/syfo/api/v3/domain/oppfolgingsplan/Person.kt
package no.nav.syfo.api.v3.domain.oppfolgingsplan

import java.time.LocalDateTime

data class Person(
    val navn: String = " ",
    val fnr: String?,
    val epost: String? = null,
    val tlf: String? = null,
    val sistInnlogget: LocalDateTime? = null,
    val samtykke: Boolean? = null,
    val evaluering: Evaluering? = null,
    val stillinger: List<Stilling> = ArrayList(),
)
